packages/components/src/tab-panel/index.tsx

```diff
@@ -220,6 +220,7 @@ const UnforwardedTabPanel = (
 							) }-view` }
 							render={
 								<Button
+									__next40pxDefaultSize
```

Comment:
It's a bit of a bummer that this prop is a no-op here due to the style override.